src/Serokell/Data/Variant/Variant.hs

```diff
@@ -66,7 +61,7 @@ instance IsString Variant where
 
 instance IsList Variant where
     type Item Variant = Variant
-    toList (VarList v) = toList v
+    toList (VarList v) = GHC.Exts.toList v
```

Comment:
Looks like you can use `toList` from `universum` because `universum` contains `ToList` instance for `Vector`:

https://github.com/serokell/universum/blob/d14925085b1fc7037f25542618b931d22dcf13b4/src/Universum/Container/Class.hs#L209